src/test/java/core/basesyntax/HelloWorldTest.java

```diff
@@ -1,8 +1,105 @@
 package core.basesyntax;
 
-/**
- * Feel free to remove this class and create your own.
- */
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.exception.InvalidUserException;
+import core.basesyntax.model.User;
+import core.basesyntax.service.RegistrationService;
+import core.basesyntax.service.RegistrationServiceImpl;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
 public class HelloWorldTest {
```

Comment:
The class name `HelloWorldTest` does not reflect the purpose of the tests. It should be renamed to something more descriptive such as `RegistrationServiceImplTest` to clearly indicate that it is testing the `RegistrationServiceImpl` class.